rdr_service/alembic/versions/15e05f673f5f_altering_biobank_dv_order_history.py

```diff
@@ -94,6 +94,8 @@ def upgrade_rdr():
     # Clean out the tmp_ table
     op.execute("DROP TABLE tmp_biobank_dv_order_history;")
 
+    # Removing the bkup_ table
+    op.execute("DROP TABLE bkup_biobank_dv_order_history;")
```

Comment:
I think this should be commented out, this should be a manual step.  We want to keep the backup table until we have verified everything is correct.  Once verified, we should manually drop the backup table.

Reply:
Sounds good. I commented it out.